must-gather/collection-scripts/gather_with_tls

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "insecure-skip-tls-verify: $1"
```

Comment:
`gather_without_tls` name was used earlier. Is this the correct file name?

Reply:
https://github.com/openshift/oadp-operator/blob/e4f87247e56bc78530b9c78554b5175806d00a95/must-gather/README.md#support-for-insecure-tls-connections

Reply:
This is not the file name. It is just a variable that we use to receive the cli args

Reply:
when the script gets invoked, this value gets printed on the console. It is more of output and has no specific use